lldb/source/Plugins/TypeSystem/Swift/TypeSystemSwiftTypeRef.cpp

```diff
@@ -180,149 +180,164 @@ ResolveTypeAlias(lldb_private::Module *M, swift::Demangle::Demangler &dem,
   return {n, {}};
 }
 
+swift::Demangle::NodePointer TypeSystemSwiftTypeRef::Transform(
+    swift::Demangle::Demangler &dem, swift::Demangle::NodePointer node,
+    std::function<swift::Demangle::NodePointer(swift::Demangle::NodePointer)>
+        fn) {
+  if (!node)
+    return node;
+  using namespace swift::Demangle;
+  llvm::SmallVector<NodePointer, 2> children;
+  bool changed = false;
+  for (unsigned i = 0; i < node->getNumChildren(); ++i) {
+    NodePointer child = node->getChild(i);
```

Comment:
I think `for (auto *child : *node)` works here

Reply:
Nice!